### PR TITLE
landing page content, site title

### PR DESCRIPTION
**Pull request by pnetzer**

### CODE REVIEW

File: app/views/pages/landing.html.haml [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+%hr
+.row.b-row
+  .medium-12.columns
+    %h4 License
+    %p BlueMap is free and licensed under MIT. Visit us at <a href="https://github.com/yoyostile/beacon-cms" target="_blank">GitHub</a> to get the source code!
+
+%hr
+.row.b-row
+  .medium-12.columns
+    %h4 Supprt
```
Comment (yoyostile) [inline app/views/pages/landing.html.haml]: 2c013f66b83e15710c95c8854be0877ce0b3ab41dff3c4ee2d313348099ee6aa

File: app/views/pages/landing.html.haml [hunk 2]
```diff
@@ -1,22 +1,52 @@
...
+    %p If you have any questions or suggestions, please don’t hesitate to <a href="mailto:wtf@wtf.wtf">contact us</a>!
```
Comment (yoyostile) [inline app/views/pages/landing.html.haml]: Maybe we should add a real address?

File: app/views/pages/landing.html.haml [hunk 3]
```diff
@@ -1,22 +1,52 @@
...
+    %p Android devices are supported with version 4.3 Jelly Bean and upwards.
+
+%hr
+.row.b-row
+  .medium-12.columns
+    %h4 About us
+    %p We are a group of 5 master students at the Institute for Media and Imaging Technology of Cologne University of Applied Sciences. This application arose within a one year master project with Prof. Dr. Luigi Lo Iacono. The Team Members in alphabetical order:
+    %ul
+      %li Peter L. Gorski
+      %li Johannes Heck
```
Comment (yoyostile) [inline app/views/pages/landing.html.haml:42]: I'm embarrassed to see my name here.

Comment (pnetzer) [inline app/views/pages/landing.html.haml:42]: Maybe we should use screen names.

**Pull request merged**